core/src/franchise/player/player.service.ts

```diff
@@ -301,6 +301,7 @@ export class PlayerService {
     ): Promise<MLE_Player> {
         const updatedPlayer = this.mle_playerRepository.merge(player, {
             updatedBy: "Sprocket FA Intake",
+            updatedAt: new Date(),
```

Comment:
I feel like this should be automatic, might be worth looking at the model and making sure the property is decorated correctly